action.yml

```diff
@@ -18,7 +18,7 @@ runs:
 
         # FOSSA has two kinds of API keys (aka tokens), a full-privilege key
         # and a low-privilege "push-only" key. The practical difference is that
-        # the full key provides more feedback on `fossa test` failure. We have
+        # the full key provides more feedback on `fossa test` errors. We have
```

Comment:
Do we really mean "errors" here?

Reply:
nope, this is for license compliance failure good catch